lib/awful/rules.lua

```diff
@@ -363,8 +363,8 @@ function rules.extra_properties.placement(c, value)
     local ty = type(value)
 
     local args = {
-        honor_workarea = true,
-        honor_padding  = true
+        honor_workarea = props.honor_workarea ~= false,
+        honor_padding  = props.honor_padding ~= false
```

Comment:
Nothing new in the module. I still have my `managing-clients.md.lua` doc pages, but it's incomplete. It started from the same code that ended up being `appearance.md.lua`. It parses the code (client.c and client.lua) to build a property database then add the "custom" ones.

But for now I added a new commit with a crappy header doc. It also adds all the other undocumented properties.